src/vivarium_public_health/risks/distributions.py

```diff
@@ -252,133 +252,6 @@ def __repr__(self):
         return f"DichotomousDistribution(risk={self.risk})"
 
 
-class LBWSGDistribution(PolytomousDistribution):
```

Comment:
This older implementation was left behind.  Actual one is now in the implementations subpackage.